docs/requirements.in

```diff
@@ -1,6 +1,7 @@
+MarkupSafe<2.1.0  # https://github.com/pallets/markupsafe/issues/282
```

Comment:
@ssbarnea @priyamsahoo In the other PR, I asked to add spaces around `<` to be consistent with the rest of the entries. Why was this ignored?